packages/editor/src/components/blocks/combobox/Combobox.tsx

```diff
@@ -26,8 +29,23 @@ export const ComboboxComponent: ComponentConfig<ComboboxProps> = {
   create: ({ label, value, ...defaultProps }) => ({ ...defaultInputProps, label, value, ...defaultProps }),
   fields: {
     label: { subsection: 'General', label: 'Label', type: 'text' },
-    value: { subsection: 'General', label: 'Value', type: 'textBrowser', options: { onlyListTypes: true } },
     completeMethod: { subsection: 'General', label: 'Complete Method', type: 'text' },
+    value: { subsection: 'Dynamic Options', label: 'List of Objects', type: 'textBrowser', options: { onlyListTypes: true } },
```

Comment:
?!?
this is wrong...
the value is the value like before and **not** with only list types
the autocomplete has no dynamic options as far as I know... the itemlabel and itemvalue are for the autocomplete method

Reply:
because of this it is currently not possible for you to add a browser for those two as you don't know the type from the autocomplete method

Reply:
Oh then I have misunderstood the autoComplete component. in this case I will assign the normal attributebrowser to the "value" field and a normal input field to itemValue itemLabel?

Reply:
the autocomplete function can return a list of objects and if you want to use an attribute of them as value or label then you need the itemvalue/label. 
the value holds as always the value

Reply:
All right, am I correct in assuming that I can hide the itemValue and itemLabel fields if completeMethod has no value?
Additionally I would add the properties ‘Label’, ‘Value’, ‘Dropdown’ in general and make a second section called ‘Options’ where I add CompleteMethod and InputValue/InputLabel.
![grafik](https://github.com/user-attachments/assets/eea03fb7-adfb-4a0a-b4a5-864d90af9e83)

Reply:
Yes you can hide those two if the complete method is empty. 
I would also move the dropdown button checkbox to the options section.